cluster/node-pools/master-default/userdata.yaml

```diff
@@ -711,3 +714,21 @@ write_files:
         - level: Request
           omitStages:
             - "RequestReceived"
+
+{{ if eq .Cluster.ConfigItems.enable_encryption "true" }}
+  - owner: root:root
+    path: /etc/kubernetes/config/encryption-config.yaml
+    permissions: '0400'
+    content: |
+      apiVersion: apiserver.config.k8s.io/v1
+      kind: EncryptionConfiguration
+      resources:
+        - resources:
+          - secrets
+          providers:
+          - aescbc:
+              keys:
+              - name: key1
```

Comment:
Do we want to change the key name?

Reply:
Key rotation works by adding a key entry, running it, re-creating all secrets and then removing the old key. Hence, I wanted to go with a "cattle-like" key scheme. I'll address that for the final PR.